src/event_callback.js

```diff
@@ -0,0 +1,46 @@
+import Car from "./car.js";
+import { showElements } from "./elements.js";
+import startRace from "./race.js";
+
+function checkNames(elems) {
+  let result = true;
+  elems.forEach(function (item) {
+    if (item.length > 5) {
+      result = false;
+    }
+  });
+  return result;
+}
+
+export function setNum() {
+  let userInputNum = document.getElementById("#racing-count-input").value;
+  let checkNum = num => !isNaN(num);
```

Comment:
```suggestion
  let checkNum = num => !Math.isNaN(num);
```

> isNaN() 함수는 어떤 값이 NaN인지 판별합니다. isNaN 함수는 몇몇 일반적이지 않은 규칙을 가지고 있으므로, ECMAScript 2015에서 추가한 Number.isNaN()으로 바꾸는 편이 좋을 수도 있습니다.

- https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/isNaN